### PR TITLE
Uniformize create_mirror* parameter name for views

### PR DESCRIPTION
**Pull request by thierryantoun**
This PR aims to mention the changes of [PR #6917](https://github.com/kokkos/kokkos/pull/6917) in the Kokkos documentation.

### CODE REVIEW

**Review by dalg24 — APPROVED** [review]

**Review by masterleinad — APPROVED** [review]

File: docs/source/API/core/view/create_mirror.rst [hunk 1]
```diff
@@ -164,4 +164,4 @@ Description
 
      .. important::
 
-	``arg_prop`` must not include a pointer to memory, or a label, or allow padding and ``arg_prop`` must include a memory space.
+	``arg_prop`` must not include a pointer to memory, or a label, or allow padding and ``arg_prop`` must include a memory space.
```
Comment (masterleinad) [inline docs/source/API/core/view/create_mirror.rst:167]:
```suggestion
	``arg_prop`` must not include a pointer to memory, or a label, or allow padding and ``arg_prop`` must include a memory space.
```

Comment (dalg24) [reply docs/source/API/core/view/create_mirror.rst:167]: @masterleinad what did you change?

Comment (dalg24) [reply docs/source/API/core/view/create_mirror.rst:167]: oh remove trailing whitespace that was added

Comment (dalg24) [inline docs/source/API/core/view/create_mirror.rst:167]:
```suggestion
	``arg_prop`` must not include a pointer to memory, or a label, or allow padding and ``arg_prop`` must include a memory space.
```

Comment (dalg24) [reply docs/source/API/core/view/create_mirror.rst:167]: Not sure why I can't just apply

Comment (dalg24) [reply docs/source/API/core/view/create_mirror.rst:167]: organization fork instead of user...

**Pull request merged**